.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: ci
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  run-tests:
+    name: Run test cases
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        go: [^1]
```

Comment:
the `actions/setup-go` looks like a neat tool!

Reply:
Yep. For go projects w/o the external dependency the CI files are very clean because of these actions. The envconsul one for example.